AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStore/src/HierarchicalVersionUtils.dfy

```diff
@@ -0,0 +1,128 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+include "../Model/AwsCryptographyKeyStoreTypes.dfy"
+include "Structure.dfy"
+include "KMSKeystoreOperations.dfy"
+include "KmsArn.dfy"
+
+module HierarchicalVersionUtils {
+
+  import opened StandardLibrary
+  import opened Wrappers
+  import opened Seq
+
+  import ErrorMessages = KeyStoreErrorMessages
+  import Types = AwsCryptographyKeyStoreTypes
+  import KMS = ComAmazonawsKmsTypes
+  
+  import AtomicPrimitives
+  import UTF8
+  import KMSKeystoreOperations
+  import KmsArn
+  import Structure
+  
+  function method GetMdDigestFromEC(
```

Comment:
Blocking: GetMdDigestFromEC implies this function returns a digest.
But it returns a map of <string,string>.

What does this method do?

It appears that it removes the "logical key store name"; 
which we could call the "storedBranchKeyContext",
so would "GetStoredBranchKeyContext" be a good name?